client/src/global/components/FatalError/index.js

```diff
@@ -1,84 +1,72 @@
-import { PureComponent } from "react";
+import { useRef } from "react";
 import PropTypes from "prop-types";
-import { Redirect } from "react-router-dom";
+import { Redirect, useHistory } from "react-router-dom";
 import { Helmet, HelmetProvider } from "react-helmet-async";
-import { withTranslation } from "react-i18next";
+import { useTranslation } from "react-i18next";
 import { Global as GlobalStyles } from "@emotion/react";
 import has from "lodash/has";
+import isEqual from "lodash/isEqual";
 import config from "config";
-import { fatalErrorActions, notificationActions } from "actions";
+import { notificationActions } from "actions";
 import lh from "helpers/linkHandler";
 import styles from "theme/styles/globalStyles";
 import ApiTrace from "./ApiTrace";
 import ClientTrace from "./ClientTrace";
 import * as Styled from "./styles";
 
-class FatalError extends PureComponent {
-  static propTypes = {
-    fatalError: PropTypes.shape({
-      error: PropTypes.object,
-      type: PropTypes.string
-    }).isRequired,
-    headerLineOne: PropTypes.string,
-    headerLineTwo: PropTypes.string,
-    dismiss: PropTypes.func,
-    dispatch: PropTypes.func,
-    i18n: PropTypes.object
-  };
+export default function FatalError(props) {
```

Comment:
I'm leaning toward pulling the presentational part of this component out into a separate component so we have a component which can render outside redux/router context and wrapper which needs both and handles redirects when called from within the app.